packages/camera/README.md

```diff
@@ -15,13 +15,18 @@ First, add `camera` as a [dependency in your pubspec.yaml file](https://flutter.
 
 ### iOS
 
-Add a row to the `ios/Runner/Info.plist` of your app with the key `Privacy - Camera Usage Description` and a usage description.
+Add two rows to the `ios/Runner/Info.plist`:
+
+* one with the key `Privacy - Camera Usage Description` and a usage description.
+* and one with the key `Privacy - Microphone Usage Description` and a usage description.
```

Comment:
Just chiming in here: IIRC, if the iOS app released to App Store contains any code that contains calls to, lets say Location APIs, there has to be keys present in Info.plist for that. Otherwise the app won't go through Apple's review. This is true even though the code calling that Location API is never executed - the mere existence of that call is enough.

So, if recording videos requires an Info.plist key, that has to be present for the app for it to pass Apple's review. Even though the "record video" code path is never called.

The app will still compile though, but will get stuck in the app review.

Reply:
The permission is only requested when the user begins recording the first video. The user can take photos without granting this permission.